### PR TITLE
Fixing date format on NovSentMail Sent At.

### PR DESCRIPTION
**Pull request by wilkerwma**

### CODE REVIEW

File: src/Nova/NovaSentMail.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public function fields(Request $request)
                     return trim(strip_tags($content));
                 })
                 ->alwaysShow(),
-            DateTime::make('Sent At', 'created_at')->format('M/d/Y h:mm:ss a'),
+            DateTime::make('Sent At', 'created_at')->format('M/D/Y h:mm:ss a'),
```
Comment (adammparker) [inline src/Nova/NovaSentMail.php:60]: @wilkerwma I think this change would mean we'd end up printing `Jan/Sat/2021`, I'm guessing we don't want that. Is this change related to a GH issue?

**Review by adammparker — CHANGES_REQUESTED** [review]

File: src/Nova/NovaSentMail.php (hunk 1 above)
Comment (wilkerwma) [reply src/Nova/NovaSentMail.php:60]: @adammparker agreed, the format string is incorrect.
But no worries about this PR anymore. You can close it if you like.
